### PR TITLE
Add angular material design library. Migrate componente to material.

### PR DESCRIPTION
**Pull request by YauheniyBaihot**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/src/components/basket/basket.component.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  public downloadItems() {
+    const request = new ContentDownloadLinkCreateRequest({
+      contents: this.basketItems.map(item => ({ contentId: item, outputFormatId: 'Original' }))
+    });
+
+    this.contentService.createDownloadLink(request).subscribe(data => {
+      if (data.downloadUrl) {
+        window.open(data.downloadUrl);
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/components/basket/basket.component.ts]: No need to open in a new window.

File: src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html [hunk 2]
```diff
@@ -1,19 +1,116 @@
...
+            <mat-icon *ngIf="isAscending === true">check</mat-icon>
+            <span>Ascending</span>
+          </button>
+          <button mat-menu-item [disabled]="activeSortingType === sortingTypes.relevance" (click)="isAscending = false; update();">
+            <mat-icon *ngIf="isAscending === false">check</mat-icon>
+            <span>Descending</span>
+          </button>
+          <mat-divider></mat-divider>
+          <button mat-menu-item (click)="setSortingType(sortingTypes.relevance)">
+            <mat-icon *ngIf="activeSortingType === sortingTypes.relevance">check</mat-icon>
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html]: Move icon to the end of the menu item to have a clean layout when one is checked.

File: src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html [hunk 3]
```diff
@@ -1,19 +1,116 @@
...
+          <button mat-menu-item [disabled]="activeSortingType === sortingTypes.relevance" (click)="isAscending = true; update();">
+            <mat-icon *ngIf="isAscending === true">check</mat-icon>
+            <span>Ascending</span>
+          </button>
+          <button mat-menu-item [disabled]="activeSortingType === sortingTypes.relevance" (click)="isAscending = false; update();">
+            <mat-icon *ngIf="isAscending === false">check</mat-icon>
+            <span>Descending</span>
+          </button>
+          <mat-divider></mat-divider>
+          <button mat-menu-item (click)="setSortingType(sortingTypes.relevance)">
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html]: Move the items to the Component and create an array of sorting items.

File: src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser-item/content-browser-item.component.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+  <!-- TODO: make it as img. -->
+  <div class="content-item__image-slot">
+    <div class="content-item__image-container">
+      <mat-spinner *ngIf="isLoading" [diameter]="thumbnailSize === thumbnailSizes.Small || isListView ? 50 : 100"></mat-spinner>
+      <!-- TODO: translate alt -->
+      <img *ngIf="!isLoading" class="content-item__image" [src]="thumbnailUrl" alt="There is no image." />
+    </div>
+  </div>
+  <div *ngIf="thumbnailSize !== thumbnailSizes.Small" class="content-item__title mat-title">
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser-item/content-browser-item.component.html]: dont use mat-title. Use smaller font like mat-subheading-1

File: src/picturepark-sdk-v1-angular/src/components/aggregation-list/aggregation.component.ts [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+import {
+  AggregationFilter, AggregationResult, AggregatorBase,
+  AggregationResultItem, TermsAggregator, ObjectAggregationResult
+} from '../../services/services';
+
+
+export abstract class AggregationComponent implements OnChanges {
+  // Used for performing aggregate request (autocomplete functionality).
+  @Input()
```
[the remark this answers is not in the document]
Comment (YauheniyBaihot) [reply src/picturepark-sdk-v1-angular/src/components/aggregation-list/aggregation.component.ts]: Done.

File: src/picturepark-sdk-v1-angular/src/components/basket/basket.component.ts (hunk 1 above)
Comment (YauheniyBaihot) [reply src/picturepark-sdk-v1-angular/src/components/basket/basket.component.ts]: Done

File: src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser-item/content-browser-item.component.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    <div>{{ itemModel.item?.displayValues['name'] || 'n/a'}}</div>
+    <div class="mat-subheading-2" *ngIf="isListView">{{ itemModel.item?.displayValues['list'] || 'n/a'}}</div>
+  </div>
+  <div class="content-item__actions">
```
[the remark this answers is not in the document]
Comment (YauheniyBaihot) [reply src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser-item/content-browser-item.component.html]: Done

File: src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html (hunk 2 above)
Comment (YauheniyBaihot) [reply src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html]: Done

File: src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html (hunk 3 above)
Comment (YauheniyBaihot) [reply src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser.component.html]: Done

File: src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser-item/content-browser-item.component.html (hunk 4 above)
Comment (YauheniyBaihot) [reply src/picturepark-sdk-v1-angular/src/components/content-browser/content-browser-item/content-browser-item.component.html]: Done

**Pull request merged**